README.md

```diff
@@ -43,18 +43,18 @@ And you are good to go! You can start [exploring our tutorial notebooks](https:/
 
 ### Example
 
-Example of how to use a scenario:
+To test this scenario we have available one example of a protein PDB file - [download it here](inductiva/resources/alanine.pdb).
 
 ```python
  import inductiva
 
  # Initialize the scenario
  scenario = inductiva.molecules.scenarios.ProteinSolvation(
-     protein_pdb = "protein_pdb_file.pdb"
+     protein_pdb = "protein_pdb_file.pdb",
      temperature = 300)
 
  # Run a simulation
- task = scenario.simulate(simulation_time_ns = 20)
+ task = scenario.simulate(simulation_time_ns = 0.5)
```

Comment:
Is this a small simulation? Having troubles finding a small simulation_time to not have to wait to long. When copy-pasting this will go to the default machines with 4 cores, so we should have those times in mind for people to test and experiment quickly.